site/weavedns.md

```diff
@@ -78,30 +143,31 @@ weave run --with-dns 10.2.1.4/24 -ti \
   --dns-search=weave.local ubuntu
 ```
 
-## Doing things more manually
+## <a name="local-domain"></a>Using a different local domain
 
-If you use the `--with-dns` option, `weave run` automatically supplies
-the DNS server address to the new container. And both `weave run` and
-`weave attach` register the hostname of the given container against
-the given weave network IP address.
+By default, WeaveDNS uses `weave.local.` as the domain for names on the
+Weave network. In general users do not need to change this domain, but you
+can force WeaveDNS to use a different domain by launching it
+with the `--domain` argument. For example,
 
-In some circumstances, you may not want to use the `weave`
-command. You can still take advantage of a running weaveDNS, with some
-extra manual steps.
+```bash
+$ weave launch-dns 10.2.254.1/24 --domain="mycompany.local."
+```
 
-### Using a different docker bridge
+The local domain should end with `local.`, since these names are
+link-local as per [RFC6762](https://tools.ietf.org/html/rfc6762),
+(though this is not strictly neccessary).
 
-So that containers can connect to a stable and always routable IP
-address, weaveDNS publishes its port 53 to the Docker bridge device,
-which is assumed to be `docker0`.
+## <a name="without-run"></a>Using weaveDNS without `weave run`
 
-Some configurations may use a different Docker bridge device. To
-supply a different bridge device, use the environment variable
-`DOCKER_BRIDGE`, e.g.,
+If you use the `--with-dns` option, `weave run` automatically supplies
+the DNS server address to the new container. And both `weave run` and
```

Comment:
This 'And' is confusing, because it implies that `run` and `attach` only register if `--with-dns` has been supplied, when in fact they are othogonal.